wallet/psbt.go

```diff
@@ -43,7 +44,18 @@ import (
 // responsibility to lock the inputs before handing the partial transaction out.
 func (w *Wallet) FundPsbt(packet *psbt.Packet, keyScope *waddrmgr.KeyScope,
 	minConfs int32, account uint32, feeSatPerKB btcutil.Amount,
-	coinSelectionStrategy CoinSelectionStrategy) (int32, error) {
+	coinSelectionStrategy CoinSelectionStrategy, optFuncs ...TxCreateOption) (int32, error) {
+
+	opts := defaultTxCreateOptions()
```

Comment:
This logic is the same in CreateSimpleTx. Maybe we can put this logic in a function but IMO it may decrease the readability